### PR TITLE
FIX improve test case

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

**Comment by fgalan** [comment]
Related PR https://github.com/telefonicaid/iotagent-node-lib/pull/1567

File: test/unit/ngsiv2/ngsiService/active-devices-test.js [hunk 1]
```diff
@@ -905,24 +905,27 @@ describe('NGSI-v2 - Active attributes test', function () {
...
+                type: 'aTypeProvidedByIoTACodeCallingUpdateOnLib3',
                 value: 'measIoTA'
             }
         ];
 
         beforeEach(function (done) {
             nock.cleanAll();
 
+            // Note that in the case of measure_id and measure_type the type provided by the IOTA when calling iotAgentLib.update()
+            // is used (thus ignoring the one of the StupidDevice group) but in the case of measIoTA the type provided in the
```
Comment (fgalan) [inline test/unit/ngsiv2/ngsiService/active-devices-test.js:927]:
```suggestion
            // is used (thus ignoring the one of the StupidDevice group for id or type, which is 'text') but in the case of measIoTA the type provided in the
```

File: test/unit/ngsiv2/ngsiService/active-devices-test.js [hunk 2]
```diff
@@ -905,24 +905,27 @@ describe('NGSI-v2 - Active attributes test', function () {
...
                 value: 'measIoTA'
             }
         ];
 
         beforeEach(function (done) {
             nock.cleanAll();
 
+            // Note that in the case of measure_id and measure_type the type provided by the IOTA when calling iotAgentLib.update()
+            // is used (thus ignoring the one of the StupidDevice group for id or type, which is 'text') but in the case of measIoTA the type provided in the
+            // provising (String) is used
```
Comment (fgalan) [inline test/unit/ngsiv2/ngsiService/active-devices-test.js:928]:
```suggestion
            // is used (thus ignoring the one of the StupidDevice group for id or type, which is 'text') but in the case of measIoTA the type provided in the
            // provisioning ('String') is used
```

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**